todo-mvc/package.json

```diff
@@ -11,6 +11,10 @@
     "url": "https://github.com/dojo/examples.git"
   },
   "license": "BSD-3-Clause",
+  "engines": {
```

Comment:
Just a note, this is not currently part of our `dojo2-package-template` though it could/should be.  We should open an issue there to track adding this.